code/go/0chain.net/conductor/README.md

```diff
@@ -1,10 +1,506 @@
+(cd 0dns && git apply ../0chain/docker.local/bin/conductor/0dns-local.patch)
+```
+9. Patch blobbers
+```sh
+(cd blobber && git apply --check ../0chain/docker.local/bin/conductor/blobber-tests.patch)
+(cd blobber && git apply ../0chain/docker.local/bin/conductor/blobber-tests.patch)
+```
+10. Add `~/.zcn/config.yaml` as follows
+```yaml
+block_worker: http://127.0.0.1:9091
+signature_scheme: bls0chain
+min_submit: 50
+min_confirmation: 50
+confirmation_chain_length: 3
+max_txn_query: 5
+query_sleep_time: 5
+```
+11. Apply if on Ubuntu 18.04
+
+https://github.com/docker/for-linux/issues/563#issuecomment-547962928
+
+The bug in Ubuntu 18.04 related. It relates to docker-credential-secretservice
+package required by docker-compose and used by docker. A docker process
+(a build, for example) can sometimes fails due to the bug. Some tests have
+internal docker builds and can fail due to this bug.
+
+12. Run blobber tests
+```sh
+(cd 0chain && ./docker.local/bin/start.conductor.sh blobber-1)
+(cd 0chain && ./docker.local/bin/start.conductor.sh blobber-2) (edited)
+```
+
+## Updating conductor tests
+
+### Updating the tests
+
+To add more test, simply create new test cases and add them to existing or new set. 
+Then be sure to enable the test set if creating a new one.
+
+### Temporarily disabling tests
+To test a specific test, simply comment out the other tests on `enable` part of the conductor test yaml.
+
+For example, only run `Miner down/up`
+
+```yaml
+enable:
+  - "Miner down/up"
+#  - "Sharder down/up"
+#  - "All miners down/up"
+#  - "All sharders down/up"
+#  - "All nodes down/up"
+```
+
+### Common directive properties
+- `timeout` - All commands support a timeout out of the box. Valid values in time duration format (eg. `1s` for 1 second, `10m` for 10 minutes). The default is 2 minutes.
+- `must_fail` - Whether the test should fail if the command throw an error. By default, built-in commands have `must_fail` as false. Custom commands however is configurable with`should_fail` when declared.
+
+### Supported directives
+
+#### Built-in directives
+1. **common setups**
+- `set_monitor` - initiate the node from where blockchain events will be accepted
+- `cleanup_bc` - stop all nodes, reset rounds, and clean up data using `cleanup_command` defined on `conductor.config.yaml`
+  
+2. **common nodes control**
+- `start` - starts the list of nodes. The start script used is defined on `conductor.config.yaml`
+- `stop` - stops the list of nodes. The stop script used is defined on `conductor.config.yaml`
+- `start_lock` - starts the list of nodes but lock them such that the nodes do nothing (eg. does not sign, does not generate blocks)
+- `unlock` - update the state of the list of nodes to be no longer locked
+  
+3. **wait for an event of the monitor**
+- `wait_view_change` - wait until a view change occurred
+  - properties
+    ```yaml
+    # name the round of this view change.
```

Comment:
Other `# comments` start with a capital, probably should be consistant `name` -> `Name`. Or make everything lowercase.